src/Symfony/Component/HttpKernel/DependencyInjection/FragmentRendererPass.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+
+/**
+ * Adds services tagged kernel.fragment_renderer as HTTP content rendering strategies.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class FragmentRendererPass implements CompilerPassInterface
+{
+    private $handlerService;
+    private $fragmentTag;
+
+    /**
+     * @param string $handlerService Service name of the fragment handler in the container
+     * @param string $fragmentTag    Tag name used for fragments
+     */
+    public function __construct($handlerService = 'fragment.handler', $fragmentTag = 'kernel.fragment_renderer')
```

Comment:
the argument name should be ``rendererTag`` or ``fragmentRendererTag``. Services tagged with this tag are renderers, not fragments